Ravel-Creators/Assets/Ravel-Creators/Scripts/Editor/Environment/Environment.cs

```diff
@@ -0,0 +1,105 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Base.Ravel.CustomAttributes;
+using Base.Ravel.Users;
+using UnityEngine;
+
+/// <summary>
+/// Data class for environments, this data is pulled from the server and put into a scriptable object as container class.
+/// </summary>
+[Serializable]
+public class Environment
+{
+    [ReadOnly]
+    public string environmentUuid;
+    
+    [ReadOnly]
+    public new string name;
```

Comment:
new?

Reply:
Name already exists, this prevents unity from throwing a warning for a duplicate varname